src/widgets/Tabs/PatientEdit.js

```diff
@@ -65,26 +65,28 @@ const PatientEditComponent = ({
   );
 
   return (
-    <FlexView style={pageTopViewContainer}>
-      <AfterInteractions placeholder={null}>
-        <FlexRow style={{ marginBottom: 7 }} justifyContent="flex-end">
-          <VaccinePrescriptionInfo />
-        </FlexRow>
-
-        <View style={localStyles.container}>
-          <View style={localStyles.formContainer}>
-            <FormControl
-              showCancelButton={false}
-              showSaveButton={false}
-              inputConfig={getFormInputConfig('patient', currentPatient)}
-              shouldAutoFocus={false}
-            />
-          </View>
-
-          <View style={localStyles.verticalSeparator} />
-          {surveySchema && surveyFormData && (
-            <View style={localStyles.formContainer}>
-              <View style={localStyles.verticalSeparator} />
+    <FlexView style={{ ...pageTopViewContainer, paddingVertical: 0 }}>
```

Comment:
looks better ( to me ) with padding
```suggestion
    <FlexView style={{ ...pageTopViewContainer }}>
```